bin/verwijderdata.php

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/php5
```

Comment:
php5?

Reply:
We moeten dit ook in profielveranderingen hangen.

Reply:
Het hangt al in profielveranderingen

Reply:
Ik kijk met m'n neus